webapp/src/components/code_review/index.tsx

```diff
@@ -0,0 +1,32 @@
+import {connect} from 'react-redux';
+
+import {GlobalState} from 'mattermost-redux/types/store';
+import {getTheme} from 'mattermost-redux/selectors/entities/preferences';
+
+import CodeReview from './code_review';
+import { Post } from 'mattermost-redux/types/posts';
+import { makeGetPostsForThread } from 'mattermost-redux/selectors/entities/posts';
+import { Props } from 'components/code_review/code_review';
+
+const mapStateToProps = (state: GlobalState, ownProps: Omit<Props, 'comments'>) => {
+    const comments: {[key:string]: Post[]} = {};
+    const commentRegex = new RegExp(`^\\\*\\\*${ownProps.fileInfo.name}:(\\d+):\\\*\\\*`);
+    const posts = makeGetPostsForThread()(state, {rootId: ownProps.post.id});
+    for (let post of posts) {
```

Comment:
0/5 This can/should be using `const` since we're not reassigning inside of the loop

```suggestion
    for (const post of posts) {
```